docs/lightning/release-notes.md

```diff
@@ -5,11 +5,14 @@ title: Release Notes
 
 ### Important Notes
 
+#### Salesforce Enhanced Domains
+If you enable [enhanced domains](https://help.salesforce.com/s/articleView?id=sf.domain_name_enhanced_enable.htm&type=5) for your Salesforce organization, you must remember to update the new syntax for all of your domains that have been allow-listed in the Amazon Connect AWS Console.
```

Comment:
Rather than specifying just to allowlist the domains, can have a more generic message like:

Salesforce, as of the enhanced domains feature in the x release, is making changes to Salesforce instance domains. Once this feature is enabled, you must migrate the CTI adapter to using these new domains. Additional details will be added shortly.

(I'm going to replace that last line w/ a document that I'm going to write on the migration process)

Reply:
Is the below statement good enough? (Is the version no correct?)

Salesforce is making changes to the instance domains on account of the enhanced domains feature in the 5.19 release. Once this feature is enabled , you must migrate the CTI adapter to using these new domains. Additional details will be added shortly.